src/hooks/usePrefersDarkMode.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+
+export function usePrefersDarkMode() {
+  const [value, setValue] = useState(true)
+
+  useEffect(() => {
+    const mediaQuery = window.matchMedia('(prefers-color-scheme: dark)')
+    setValue(mediaQuery.matches)
+
+    const handler = () => setValue(mediaQuery.matches)
+    mediaQuery.addEventListener('change', handler)
+    return () => mediaQuery.removeEventListener('change', handler)
+  }, [])
+
+  return value
+}
```

Comment:
instead of this mumbo-jumbo, just use [next-themes](https://www.npmjs.com/package/next-themes) to reduce code significantly.

I used to write my own code for dark mode too until I found **next-themes**

nice site bdw @jeffjadulco 👏

Reply:
Thanks for the suggestion! I will definitely check this out 😊